doc/sg/c_cpp_coding_style.md

```diff
@@ -86,6 +86,25 @@ void sort(int array[], size_t size){
 }
 ```
 
+```c
+/**
+ * Boot data stored in the flash info partition.
+ */
+typedef struct boot_data {
+  /**
+   * CRC32 checksum of boot data.
+   *
+   * The region covered by this checksum starts immediately after this field and
+   * ends at the end of the entry.
+   */
+  uint32_t checksum;
+  /**
+   * Boot data identifier.
+   */
+  uint32_t identifier;
+} boot_data_t;
+```
+
```

Comment:
I thought this PR had the goal of clarifying how to document structs and enums, isn't it @luismarques.